yolo/export_yolov8.py

```diff
@@ -59,7 +57,7 @@ def export_onnx(self):
                         training=torch.onnx.TrainingMode.EVAL,
                         do_constant_folding=True,
                         input_names=['images'],
-                        output_names=['output1_yolov6r2', 'output2_yolov6r2', 'output3_yolov6r2'],
+                        output_names=["output{i+1}_yolov6r2" for i in range(self.num_branches) ],
```

Comment:
There is missing `f` in the formatted string.
@richard-xx, could you please add f before the first double quote, so it'll be like this: `output_names=[f"output{i+1}_yolov6r2" for i in range(self.num_branches) ]`?